mubench.reviewsite/tests/ReviewControllerTest.php

```diff
@@ -54,4 +54,60 @@ function test_stores_violation_types()
         self::assertEquals("missing/call", $violation_types[0]["name"]);
     }
 
+    function test_determine_next_misuse()
+    {
+        $this->setupSomeRunWithThreeMisusesAndTwoReviews();
+
+        list($previous_misuse, $next_misuse, $next_reviewable_misuse, $misuse)  = $this->reviewController->determineNavigationTargets(Misuse::all(), '-p-', '-v-', '0', Reviewer::find(1));
+        self::assertEquals('2', $previous_misuse->misuse_muid);
+        self::assertEquals('1', $next_misuse->misuse_muid);
+        self::assertEquals('2', $next_reviewable_misuse->misuse_muid);
+        self::assertEquals('0', $misuse->misuse_muid);
+    }
+
+    function test_determine_next_is_previous_misuse()
+    {
+        $this->setupSomeRunWithThreeMisusesAndTwoReviews();
```

Comment:
This helper suggests that it is irrelevant which misuses you create and which of them have reviews. In fact, the name is even wrong, because you're creating three misuses two of which have two reviews each. Regardless, it is very important for the test which misuses have how many reviews, because that determines the outcome of the logic (w.r.t next reviewable). Therefore, please explicitly create the misuses and the reviews in the test methods. I'm sure you don't actually need the same data setup in both tests.